src/Server.js

```diff
@@ -704,9 +720,13 @@ class Server {
         }
         // Spawn player safely (do not check minions)
         var cell = new Entity.PlayerCell(this, player, pos, size);
-        if (this.willCollide(cell) && !player.isMi)
-            pos = this.randomPos(); // Not safe => retry
-        this.addNode(cell);
+        if (!player.isMi) {
+            this.addNode(cell);
+        }
+        else {
+            this.safeSpawn(cell);
+        }
+        
```

Comment:
This change introduces a bug where players get one shot at a safe start position, then they start playing at that position, whether safe or not. And minions actually get spawned safely.
That is the exact opposite logic of that which was intended here.

Reply:
Oops, you're right. Line 723 should be
```js
        if (player.isMi) {
```
(exclamation mark removed)